NEON support causes build failures with GCC 12, so just disable it.

--- a/Quake/mathlib.h
+++ b/Quake/mathlib.h
@@ -157,9 +157,6 @@
 #define USE_SIMD
 #define USE_SSE2
 #include <emmintrin.h>
-#elif defined(__aarch64__) && defined(__ARM_NEON)
-#define USE_SIMD
-#define USE_NEON
 #endif
 
 /*==========================================================================*/
